Packs/SecurityScorecard/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,18 @@
+#### Integrations
+
+##### SecurityScorecard
+
+-   Updated the Docker image to: _demisto/python3:3.11.10.111526_.
+-   Added new commands: _!securityscorecard-company-events-get_ , _!securityscorecard-company-findings-get_ and _!securityscorecard-issue-metadata_.
+
+<!--
```

Comment:
Please remove comments and grayed out text

Reply:
The commented out lines are a way to ignore changes that should not be in the release notes. 
It's in your best practices here https://xsoar.pan.dev/docs/documentation/release-notes#excluding-items